bot/scenes.js

```diff
@@ -257,9 +257,8 @@ const addFiatAmountWizard = new Scenes.WizardScene(
   async (ctx) => {
     try {
       const { bot, order, caller } = ctx.wizard.state;
-      const currency = getCurrency(order.fiat_code);
-      const action = order.type === 'buy' ? 'recibir' : 'enviar';
-      const currencyName = (!!currency && !!currency.name_plural) ? currency.name_plural : order.fiat_code;
```

Comment:
This doesn't need to be removed, if it's a known currency we can show the plural, if it is not we show the code, better UX

Reply:
The plural depends on language! So better show internationally recognized EUR than 'euros' when writing in Russian, for example.

Reply:
There is a lot of languages that have plural, most of the currencies are tied to a languange, each currency have it own plural field, if the language doesn't have plural in that language the plural field should be the same than non plural, as you fix it for russian, this is the reason we need to keep showing that way

Reply:
I respectfully disagree. Plural only works for one language per each currency. My solution works for all possible language-currency pairs.